src/fidesops/ops/schemas/saas/strategy_configuration.py

```diff
@@ -87,6 +91,33 @@ class CursorPaginationConfiguration(StrategyConfiguration):
     field: str
 
 
+class ApiKeyAuthenticationParameter(QueryParam):
+    method: Optional[List[HTTPMethod]]
+
+
+class ApiKeyAuthenticationConfiguration(StrategyConfiguration):
+    """
+    API key parameter to be added in as a header or query param
+    """
+
+    header: Optional[List[ApiKeyAuthenticationParameter]]
```

Comment:
What are your thoughts on changing header and query_param to plural? This way it can be consistent with the way headers and query_params are defined in the requests and it better reflects the fact that you can include a list of both.